packages/domains-table/src/utils/is-free-url-domain-name.ts

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
Is there any limitation for us [to reuse that original code](https://github.com/Automattic/wp-calypso/blob/19451117505be00b465f99767aa8b633fa0c6b28/client/lib/domains/utils/is-free-url-domain-name.ts)? If so, can we refactor to use one single source?

Reply:
I had this lint error, so I think is better to keep it inside the domains-table package
`error  'calypso/lib/domains/utils' import is restricted from being used by a pattern  no-restricted-imports`

Reply:
Do you think ignoring the lint could be a better approach?

Reply:
Oh, I see we're editing package files...

Maybe we could do the other way around, by [removing that original file](https://github.com/Automattic/wp-calypso/blob/HEAD/client/lib/domains/utils/is-free-url-domain-name.ts#L0-L1) and changing the imports [like this one ](https://github.com/Automattic/wp-calypso/blob/fix/hide-change-site-name-on-non-free-domains/client/my-sites/domains/domain-management/controller.jsx#L3) to `import { isFreeUrlDomainName } from '@automattic/domains-table/src/utils/is-free-url-domain-name';`?

My main concern is that it seems we need to maintain those domains updated as we add new free sub-domains and having one place only would help with that.

Reply:
Thanks for your update @paulopmt1, it looks great!